### ISSUE TITLE
notebooks' save_images helper broken on windows 

### ISSUE THREAD

**Issue by cchriste** [SCIInstitute/ShapeWorks#1112]
This is used in getting-started-with-grooming-segmentations and probably others.
- itk doesn't like linux paths
- the shapenames are themselves directories, so just creating outdir isn't enough

It can be fixed by updating save_images helper (eventually this will be added to sw module):
```python
    ...
    for curImg, curName in zip(swImageList, swImageNames):
        filename = Path(outDir + curName + '.' + extension)
        if verbose:
            print('Writing: ' + str(filename))
        #this fixes it (but could be cleaner):
        if not os.path.exists(os.path.dirname(filename)):
        os.makedirs(os.path.dirname(filename))
        curImg.write(str(filename), compressed=compressed) 
```

**Comment by cchriste** [comment]
Didn't quite fix it, but #1141 should do the trick.

**Comment by akenmorris** [comment]
Merged
